app/upgrades/v5/upgrades_test.go

```diff
@@ -124,3 +128,76 @@ func (suite *UpgradeTestSuite) TestScheduledUpgrade() {
 		})
 	}
 }
+
+func (suite *UpgradeTestSuite) TestAirdropHandle() {
+
+	testCases := []struct {
+		name     string
+		original []bool
+		expected []bool
+	}{
+		{
+			"EVM-IBC claimed",
+			[]bool{false, false, true, true},
+			// Swap ibc<->vote
```

Comment:
move the comment to description (line 140)

Reply:
+1, it will make the test cases names more descriptive